services/examples/openlayers-wms-static/index.html

```diff
@@ -91,6 +91,7 @@
 
             console.log("Creating layer with workflow id:", workflow_id);
 
+            // TODO: add authentication header
```

Comment:
Is it now crashing?

Reply:
it won't work but it doesn't work in master right now either because of tokio